callouts/python/extproc/example/add_body/service_callout_example.py

```diff
@@ -21,22 +21,41 @@ class CalloutServerExample(callout_server.CalloutServer):
   """Example callout server.
 
   Provides a non-comprehensive set of responses for each of the possible
-  callout interactions.
```

Comment:
nit: I believe callout is correct in this case. https://www.merriam-webster.com/grammar/how-to-use-plural-s-and-es-exceptions-grammar#:~:text=Most%20compounds%20that,than%20this%2C%20though.